modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -279,23 +332,26 @@ public function createOrderTableIfNotExist() {
 							$table->float('value');
 							$table->integer('bits');
 							$table->integer('bits_payed');
-							$table->text('flyp_id');
+							$table->string('blockonomics_currency');
 						}
 				);
 			} catch (\Exception $e) {
 					echo "Unable to create blockonomics_bitcoin_orders: {$e->getMessage()}";
 			}
-		}else if(!Capsule::schema()->hasColumn('blockonomics_bitcoin_orders', 'flyp_id')){
-			 Capsule::schema()->table('blockonomics_bitcoin_orders', function($table){
-				$table->text('flyp_id');
-			 });
+		}else{
+			if(!Capsule::schema()->hasColumn('blockonomics_bitcoin_orders', 'blockonomics_currency')){
+				 Capsule::schema()->table('blockonomics_bitcoin_orders', function($table){
+					$table->string('blockonomics_currency');
```

Comment:
We may need to set default value of this to 'btc' to take care of existing invoices/ invoices with ongoing payments. 
Also lets remove flyp_id column if it exists

Reply:
Removed flyp_id in 04ed4ca